function-template/Dockerfile

```diff
@@ -8,10 +8,10 @@ COPY . .
 
 RUN echo -n ${HANDLER} > /tmp/handler && \
     [ -n "${HANDLER}" ] || \
-    ([ "$(ls *.clj | wc -l)" == "1" ] || (echo "Exactly 1 source file needed when HANDLER is empty" >&2 && exit 1)) && \
+    (([ "$(ls *.clj | wc -l)" -eq "1" ] || (echo "Exactly 1 source file needed when HANDLER is empty" >&2 && exit 1)) && \
```

Comment:
What's wrong with `==`?

Reply:
In this specific case either `==` or `-eq` works but `-eq` is better for numeric comparisons. This also makes it more consistent with our other base images.

Reply:
Thanks! Merge away!